openfecwebapp/data_prep/financial_summaries.py

```diff
@@ -1,11 +1,43 @@
 import re
 import locale
 
+from marshmallow import Schema, fields, pre_load
+
 from openfecwebapp.api_caller import load_cmte_financials
 from openfecwebapp.data_prep.shared import committee_type_map
 
+
 locale.setlocale(locale.LC_ALL, '')
 
+class Currency(fields.Decimal):
+    def _serialize(self, value, attr, obj):
+        if value is None:
+            return fields.Decimal('0')
+
+        return locale.currency(value, grouping=True)
+
+class CommitteeFinancials(Schema):
+    total_receipts = Currency(attribute='receipts')
```

Comment:
Possibly beyond the scope of this PR, but I'm not sure why we're going to the effort of remapping these keys. If `total_cash` is a better name than `cash_on_hand_end_period`, why not have the API call this field `total_cash`? I really like the direction this patch is taking, but it could be even simpler if we could harmonize variable names across repos.